pep-0680.rst

```diff
@@ -332,14 +332,17 @@ support could be added later in a backward-compatible way.
 Removing support for ``parse_float`` in ``tomllib.load[s]``
 -----------------------------------------------------------
 
-This option is not strictly necessary, since TOML floats are "IEEE 754 binary64
-values", which is equivalent to a Python ``float`` on most architectures.
-However, parsing floats differently, such as to ``decimal.Decimal``, allows
-users extra precision beyond that promised by the TOML format. In the
-author of ``tomli``'s experience, this is particularly useful in scientific and
-financial applications. This is also useful for other cases that need greater
-precision, or where end-users include non-developers who may not be aware of
-the limits of binary64 floats.
+This option is not strictly necessary, since TOML floats should be implemented
+as "IEEE 754 binary64 values", which is equivalent to a Python ``float`` on most
```

Comment:
Could IEEE 754... be linked to the specification / definition text?

Reply:
Sure, but does an official link to a free copy exist? I found https://standards.ieee.org/standard/754-2019.html but it seems the standard is behind a paywall...

Reply:
The original [IEEE 754-1985](https://ieeexplore.ieee.org/document/30711) _may_ be [open access](https://ieeexplore.ieee.org/stamp/stamp.jsp?tp=&arnumber=30711) from the IEEE site, but I can't tell because I'm on a NASA/University network and get access to all the versions through that, and it also refers to `binary64` as `double` unlike later verisons. Otherwise, it may not be authoritative, but there's always the [Wikipedia article on binary64](https://en.wikipedia.org/wiki/Double-precision_floating-point_format#IEEE_754_double-precision_binary_floating-point_format:_binary64)

Reply:
The 1985 version also seems to require sign-in or purchase. Not sure if a link to Wikipedia is better than no link. I think personally I'd just leave this as is (no link).

Reply:
Ahh, not ideal. Probably right that no link is better. Thanks both for looking into it.

A